Sources/Nimble_Snapshots/HaveValidSnapshot.swift

```diff
@@ -44,10 +47,10 @@ public class FBSnapshotTest: NSObject {
                                referenceDirectory: String,
                                tolerance: CGFloat,
                                perPixelTolerance: CGFloat,
-                               filename: String,
+                               filename: FileString,
```

Comment:
What's the reason for using `FileString` instead?

Reply:
`FileString` is defined by `Nimble`. It's either `StaticString` for swift package or `String` so I had to adopt it